internal/injector/injector.go

```diff
@@ -64,6 +65,13 @@ type (
 		Filename string
 	}
 
+	parameters struct {
```

Comment:
That name is a little opaque... I'd prefer something a little more specific?

Reply:
Since the return value was named `result` I figured this was in the theme lol.

Reply:
Touché!